sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -7727,7 +7727,7 @@
                           "value": ""
                         }
                       ],
-                      "content": "<h4 class=\"category-header-medium-small\">Documents accepted:</h4>\n<ul>\n  <li>Valid confirmation of permanent residence (<a href=\"https://www.cic.gc.ca/english/helpcentre/answer.asp?qnum=1446&amp;top=14#confirmation\">IMM5292)</a> <strong>or</strong></li>\n  <li>Valid <a href=\"https://www.cic.gc.ca/english/helpcentre/answer.asp?qnum=1446&amp;top=14#pr-card\">permanent resident card</a> <strong>or</strong></li>\n  <li>Valid verification of status document</li>\n</ul>\n",
+                      "content": "<h4 class=\"category-header-medium-small\">Documents accepted:</h4>\n<ul>\n  <li>Valid Confirmation of SIN document ; and</li>\n  <li>Notice of Decision (issued by Immigration and Refugee Board) or</li>\n  <li>Verification of Status document (IMM5009) or</li>\n  <li>Confirmation of Protected Persons Status document (IMM5292)</li>\n</ul>",
```

Comment:
Even being defined in the business AC, the white space before the ";" is not correct "Valid Confirmation of SIN document ; and".
Feel free to ask business.

Reply:
Thanks for catching that, you're right I think that's a typo in the AC. Updated.